tests/test_coin_combination.py

```diff
@@ -1,3 +1,19 @@
 from app.coin_combination import get_coin_combination
 
-# write your tests here
+
+def test_type():
```

Comment:
It seems that this test is unnecessary. If some of these assertions fail you will see the corresponding error in the second and third tests.